react-native/react/native/pinentry-listeners.desktop.js

```diff
@@ -24,13 +27,28 @@ export default {
     }
 
     var props = payload.pinentry
-    // Here's where we should spawn a BrowserWindow with the Pinentry component,
-    // passing in props.  For now, it's stubbed out.
-    const reply = {
-      'passphrase': 'fooBARbaz',
-      'storeSecret': false
-    }
-    response.result(reply)
-    console.log('Sent passphrase back')
+    console.log(props)
+    var pinentryWindow = new BrowserWindow({
+      width: 500, height: 300,
+      //resizable: false,
+      fullscreen: false
+    })
+    pinentryWindow.hide()
+    pinentryWindow.loadUrl(`file://${__dirname}/pinentry.wrapper.html`)
```

Comment:
Not sure if this is helpful, but I'll mention it just in case:

 you can pass extra data through the hash (e.g. `.../foo.html#moreDataHere`)

maybe you can serialize props and pass them with the url?